### ISSUE TITLE
 Chronograf (1.7.14) undefined is not an object (evaluating 'a.author.name')

### ISSUE THREAD

**Issue by jwrw** [influxdata/chronograf#5421]
What browser are you using (name and version)?
Safari 13.0.5

What operating system are you using?
macOS 10.13.6

Please describe what you were trying to do when you encountered this error:
Amending an alert definition - pressed save.  Previously had used Cmnd-Click to open the alert definition in a second tab so I could compare the definitions of two alerts.???

A 'save' seemed to hang - so went back to main dashboard and got this...

```
http://192.168.2.80:8888/src.d4e31fda.js:2363:384
map@[native code]
l@http://192.168.2.80:8888/src.d4e31fda.js:2363:313
sa@http://192.168.2.80:8888/src.d4e31fda.js:717:55870
$a@http://192.168.2.80:8888/src.d4e31fda.js:717:68129
oo@http://192.168.2.80:8888/src.d4e31fda.js:717:91057
uo@http://192.168.2.80:8888/src.d4e31fda.js:717:91444
Yo@http://192.168.2.80:8888/src.d4e31fda.js:717:98526
$o@http://192.168.2.80:8888/src.d4e31fda.js:717:97906
jo@http://192.168.2.80:8888/src.d4e31fda.js:717:96927
ho@http://192.168.2.80:8888/src.d4e31fda.js:717:95740
enqueueSetState@http://192.168.2.80:8888/src.d4e31fda.js:717:47536
http://192.168.2.80:8888/src.d4e31fda.js:708:1711
http://192.168.2.80:8888/src.d4e31fda.js:802:5035
[native code]
h@http://192.168.2.80:8888/src.d4e31fda.js:761:1186
http://192.168.2.80:8888/src.d4e31fda.js:1140:156
http://192.168.2.80:8888/src.d4e31fda.js:1156:260
http://192.168.2.80:8888/src.d4e31fda.js:1138:1220
http://192.168.2.80:8888/src.d4e31fda.js:1164:1354
L@http://192.168.2.80:8888/src.d4e31fda.js:5:4405
http://192.168.2.80:8888/src.d4e31fda.js:5:4190
c@http://192.168.2.80:8888/src.d4e31fda.js:1164:363
promiseReactionJob@[native code]
```

**Comment by russorat** [comment]
@jwrw thanks for opening the issue. We will take a look.

**Comment by glinton** [comment]
This should be fixed by #5420 and will be available in 1.8.1 (or is available immediately with a [nightly build](https://portal.influxdata.com/downloads/)). Comment if that is not the case and we'll re-open this)
